addons/mail/static/src/new/core/persona_service.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module */
+
+import { Persona } from "@mail/new/core/persona_model";
+import { createLocalId } from "./thread_model.create_local_id";
+
+export class PersonaService {
+    constructor(env, services) {
+        this.env = env;
```

Comment:
I would like to remove env eventually. Can we avoid storing it when it is not used?